lib/api/controllers/securityController.js

```diff
@@ -385,3 +388,31 @@ module.exports = function SecurityController (kuzzle) {
   };
 
 };
+
+/*eslint vars-on-top: 0*/
+
+var createOrReplaceRole = function (requestObject, context, opts) {
+  this.pluginsManager.trigger('security:createRole', requestObject);
```

Comment:
Either the `security:createOrReplace*` are removed, and they should be removed from `config/hooks.js` too, or this hook must depend on the `opts` content